deepmd/pt/model/model/model.py

```diff
@@ -1,148 +1,8 @@
 # SPDX-License-Identifier: LGPL-3.0-or-later
-import logging
-import os
-
-import numpy as np
 import torch
 
-from deepmd.pt.utils import (
-    env,
-)
-from deepmd.pt.utils.stat import (
-    compute_output_stats,
-)
-
-log = logging.getLogger(__name__)
-
 
 class BaseModel(torch.nn.Module):
     def __init__(self):
         """Construct a basic model for different tasks."""
         super().__init__()
-
-    def compute_or_load_stat(
```

Comment:
Should keep this method as the interface, if not all `BaseModel` is `DPAtomicModel`.

Reply:
Yes, I agree, but the `compute_or_load_stat` here was previously designed for models with descriptors, while `BaseModel` may not necessarily have descriptors. However, `DPAtomicModel` definitely has descriptors, so this function has been moved to DPAtomicModel. But I also agree that it's good to keep a parent class function interface here, which makes it convenient for models like `PairTabModel` to inherit.